packages/ember-glimmer/lib/index.js

```diff
@@ -1,23 +1,19 @@
 /**
-  [Glimmer](https://github.com/tildeio/glimmer) is a [Handlebars](http://handlebarsjs.com/)
-  compatible templating engine used by Ember.js.
-  Any valid Handlebars syntax is valid in an Ember template.
+  [Glimmer](https://github.com/tildeio/glimmer) is a templating engine used by Ember.js that is compatible with a subset of the [Handlebars](http://handlebarsjs.com/) syntax.
 
   ### Showing a property
 
   Templates manage the flow of an application's UI, and display state (through
   the DOM) to a user. For example, given a component with the property "name",
   that component's template can use the name in several ways:
 
-  ```javascript
-    // app/components/person.js
+  ```app/components/person.js
```

Comment:
It should! http://emberjs.com/api/classes/Ember.Route.html#method_model https://github.com/emberjs/ember.js/blob/v2.11.0/packages/ember-routing/lib/system/route.js#L1481